scripts/docs.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+const fs = require('fs');
+const { exec } = require('child_process');
+
+const DELETE_EMPTY_DOCS = true
+const BASE_DOCS_FOLDER = 'docs/dapp'
+const excludedFolders = ['__tests__', '__util__']
+
+const getEmptyFileString = (function () {
+    let emptyFile
```

Comment:
I don't where `emptyFile` is being written.

Reply:
changed to a simpler promise as discussed